maint/extractcvars.in

```diff
@@ -155,6 +155,8 @@ print OUTPUT_H <<EOT;
 
 #include "mpitimpl.h" /* for MPIR_T_cvar_range_value_t */
 
+#define MPIR_CVAR_ENUM_NULL  (0)
```

Comment:
@pavanbalaji I understand this feature is useful for an advanced user, who wants to provide comma separated list of algorithms. But in more general case, like you said, CVARs are meant to be used by tools, user should just the rely on json based selection. And MPITUNE cannot make use of this feature. So what problem are we trying to solve with this feature? Is it really needed?

I know @zhenggb72 and you had discussions offline as well on this, but I am not caught up yet.

Reply:
@jain-surabhi-23  I don't want to argue for this feature given my limited understanding of MPITUNE.  I'd be OK with delaying this patch till we figure that part out.  Let me rework this PR to remove that patch.

Reply:
Thanks. Delaying sounds good to me. May be we can discuss this in person when you are in Austin?